packages/perseus-core/src/widgets/sorter/sorter-util.test.ts

```diff
@@ -15,10 +15,6 @@ describe("getSorterPublicWidgetOptions", () => {
         const publicWidgetOptions = getSorterPublicWidgetOptions(options);
 
         // Assert
-        expect(publicWidgetOptions).toEqual({
-            correct: ["$0.005$ kilograms", "$15$ grams", "$55$ grams"],
-            layout: "horizontal",
-            padding: true,
-        });
+        expect(publicWidgetOptions.correct).not.toEqual(options.correct);
```

Comment:
Minor: this assertion is a little loose / nonspecific. For example, it would pass if `correct` were set to an empty array.

I might prefer something like this:

```suggestion
        expect(publicWidgetOptions.correct).not.toEqual(options.correct);        
        expect(new Set(publicWidgetOptions.correct)).toEqual(new Set(options.correct));
```

That ensures that all the original elements are still there, but their order is different.

I wrote [a blog post about this topic](https://bensguide.substack.com/p/testing-randomness-with-loose-assertions)!

Reply:
Just make sure we keep the `.not` in that assertion. :) 

```
expect(new Set(publicWidgetOptions.correct)).not.toEqual(new Set(options.correct));
```

Reply:
It looks like you don't add the ".not" here because this new expect is just to confirm that the two sets have the same elements.

Reply:
@Myranae is right, we want the sets of elements to be the same, but the arrays (the order) to be different.